assets/differentialabundance_report.Rmd

```diff
@@ -405,23 +433,25 @@ The following plots show the abundance value distributions of input matrices. A
 ```
 #### Box plots
 
-```{r, echo=FALSE, results='asis'}
+```{r, echo=FALSE, results='asis', fig.height=8}
 
-ggplot_boxplot(
+p <- ggplot_boxplot(
   assay_data, 
   experiment = observations, 
   colorby = main_grouping_variable, 
   expressiontype = paste("count per", params$features_type), 
   palette = groupColorScale,
-  whisker_distance = params$exploratory_whisker_distance
+  whisker_distance = params$exploratory_whisker_distance,
+  base_size=8
 )
+p
```

Comment:
are you using `r` later on somewhere? Otherwise I would just not assign it and directly plot it, to make it a bit cleaner.